src/components/tracked-controls.js

```diff
@@ -18,7 +17,13 @@ module.exports.Component = registerComponent('tracked-controls', {
     controller: {default: 0},
     id: {type: 'string', default: ''},
     idPrefix: {type: 'string', default: ''},
-    rotationOffset: {default: 0}
+    rotationOffset: {default: 0},
+    // Arm model parameters, to use when not 6DOF. (pose hasPosition false, no position)
+    headElement: {type: 'selector'},
+    hand: {type: 'string', default: 'right'},
```

Comment:
In daydream then handedness is configured by the user. It should not be the application's job. Should the handedness be reported by the gamepad API instead of being configurable?